exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -170,35 +180,30 @@ def _pull_docker_db_images_if_necassary(self):
 
     def _prepare_db_volume(self, docker_client, db_private_network: str,
                            docker_db_image_info: ImageInfo) -> Volume:
-        db_volume_name = self._get_db_volume_name()
-        db_volume_preperation_container_name = self._get_db_volume_preperation_container_name()
-        self._remove_container(db_volume_preperation_container_name)
-        self._remove_volume(db_volume_name)
-        db_volume, volume_preparation_container = \
-            volume_preparation_container, volume_preparation_container = \
-            self._create_volume_and_container(docker_client, db_volume_name,
-                                              db_volume_preperation_container_name)
+        volume, container = self._prepare_volume(
+            docker_client,
+            self._get_db_volume_name(),
+            self._get_db_volume_preparation_container_name(),
+            remove_old_instances=True,
+        )
         try:
-            self._upload_init_db_files(volume_preparation_container,
-                                       db_private_network)
-            self._execute_init_db(db_volume, volume_preparation_container)
-            return db_volume
+            self._upload_init_db_files(container, db_private_network)
+            self._execute_init_db(volume, container)
+            return volume
         finally:
-            volume_preparation_container.remove(force=True)
+            container.remove(force=True)
 
-    def _get_db_volume_preperation_container_name(self):
-        db_volume_preperation_container_name = f"""{self.db_container_name}_preparation"""
-        return db_volume_preperation_container_name
+    def _get_db_volume_preparation_container_name(self):
+        return f"""{self.db_container_name}_preparation"""
 
     def _get_db_volume_name(self):
-        db_volume_name = f"""{self.db_container_name}_volume"""
-        return db_volume_name
+        return f"""{self.db_container_name}_volume"""
 
-    def _remove_container(self, db_volume_preperation_container_name):
+    def _remove_container(self, db_volume_preparation_container_name):
```

Comment:
Actually the old name `db_volume_preparation_container_name` indicates correctly that this parameter is the _name_ of a container (datatype string) rather than an object of type `Container`.